hack/dind-cluster.sh

```diff
@@ -131,20 +131,13 @@ function stop() {
 
   echo "Stopping dind cluster '${cluster_id}'"
 
-  local master_cid
-  master_cid="$(${DOCKER_CMD} ps -qa --filter "name=${MASTER_NAME}")"
-  if [[ "${master_cid}" ]]; then
-    ${DOCKER_CMD} rm -f "${master_cid}" > /dev/null
-  fi
+  for cid in $( ${DOCKER_CMD} ps -qa --filter "name=${MASTER_NAME}" ); do
```

Comment:
If you're doing the same thing to the master and node containers, the filter supports regex:

```sh
for cid in $( ${DOCKER_CMD} ps -qa --filter "name=${MASTER_NAME}|${NODE_PREFIX}" ); do
    #  ...
done
```